src/TransformerGo.cs

```diff
@@ -146,13 +146,27 @@ private void TransformEnumTypes(CodeModelGo cmg)
             {
                 if (em.Values.Where(v => topLevelNames.Contains(v.Name) || CodeNamerGo.Instance.UserDefinedNames.Contains(v.Name)).Any())
                 {
-                    em.HasUniqueNames = false;
+                    foreach (var v in em.Values)
+                    {
+                        v.Name = em.Name + v.Name;

```

Comment:
Presumably this introduces some breaking changes yes?

Reply:
No, I am just moving this lines from the `EnumTypeGo` file.

Reply:
Ah I see now, I like this better.  :)